orderly_web/config.py

```diff
@@ -131,16 +131,18 @@ def __init__(self, path, dat):
             dat, ["web", "auth", "montagu"])
         self.web_auth_fine_grained = config_boolean(
             dat, ["web", "auth", "fine_grained"])
-        self.web_auth_github_org = config_string(
-            dat, ["web", "auth", "github_org"], True)
-        self.web_auth_github_team = config_string(
-            dat, ["web", "auth", "github_team"], True)
 
         if not self.web_auth_montagu:
             self.web_auth_github_app = config_dict(dat, ["web", "auth",
                                                          "github_oauth"])
+            self.web_auth_github_org = config_string(
+                dat, ["web", "auth", "github_org"])
+            self.web_auth_github_team = config_string(
+                dat, ["web", "auth", "github_team"], True)
         else:
             self.web_auth_github_app = None
+            self.web_auth_github_org = None
```

Comment:
Just looking at OW on UAT and these None values are being written in to the web config as the string "None". We could either not write them in if None, or write them in as the empty string...